front/app/api/graph_data_units/useGraphDataUnitsLive.ts

```diff
@@ -8,24 +8,26 @@ import { CLErrors } from 'typings';
 const fetchGraphDataUnitsLive = <Response extends BaseResponseData>({
   resolvedName,
   props,
-}: ParametersLive) =>
-  fetcher<Response>({
+}: ParametersLive) => {
+  const _props = props as any;
```

Comment:
I left this one for another ticket https://www.notion.so/citizenlab/Better-types-for-graph-data-unit-hooks-0d35926432f443c8b047600d8a224183